apps/hash-graph/lib/graph/src/shared/identifier/time/interval.rs

```diff
@@ -49,7 +51,7 @@ impl<A> ToSchema<'_> for UnresolvedTimeInterval<A> {
                     openapi::Schema::OneOf(
                         openapi::OneOfBuilder::new()
                             .item(openapi::Ref::from_schema_name(
-                                TimeIntervalBound::<A>::schema().0,
+                                LimitedTimeIntervalBound::<A>::schema().0,
```

Comment:
The `start` property is using `TimeIntervalBound`, is that correct if the end is `Limited*`?

Reply:
@TimDiekmann can you look at this Q

Reply:
Yes, this is the intention. We don't allow `"unbounded"` upper bounds in the time axes